frontend/src/quantise-data.js

```diff
@@ -6,7 +6,20 @@ const randomIntervalMultiplier = 1 / noteInterval;
 
 function quantiseData (tubeData) {
   if (tubeData.length > 0) {
-    const quantisedData = tubeData.map((train) => {
+    const abridgedData = tubeData.map(obj => {
+      // Modify properties by removing whitespace, apostrophes, and "Underground Station"
+      return {
+        id: obj.id,
+    stationName: obj.stationName
```

Comment:
This is really great, thanks Pablo! I think we should split this out into its own function e.g. reformatStationName is all I would say - just to tidy it up a bit